minifi-bootstrap/src/main/java/org/apache/nifi/minifi/bootstrap/RunMiNiFi.java

```diff
@@ -357,8 +387,10 @@ private synchronized void saveProperties(final Properties nifiProps, final Logge
 
         try {
             final Set<PosixFilePermission> perms = new HashSet<>();
-            perms.add(PosixFilePermission.OWNER_READ);
             perms.add(PosixFilePermission.OWNER_WRITE);
+            perms.add(PosixFilePermission.OWNER_READ);
+            perms.add(PosixFilePermission.GROUP_READ);
+            perms.add(PosixFilePermission.OTHERS_READ);
```

Comment:
On further investigation, it seems like this is just the pid file (I initially assumed it was the properties file)

I'm ok with others knowing the pid since OS permissions should still keep them from doing bad things.  (Also, it's not uncommon to be able to list all processes)